src/k8s-engine/hyperkit.ts

```diff
@@ -196,11 +214,10 @@ export default class HyperkitBackend extends events.EventEmitter implements K8s.
   /** Get the IPv4 address of the VM, assuming it's already up */
   protected get ipAddress(): Promise<string> {
     return (async() => {
-      const driver = resources.executable('docker-machine-driver-hyperkit');
-      const args = [
-        '--storage-path', path.join(paths.state(), 'driver'),
+      const { driver, defaultArgs } = this.hyperkitArgs;
+      const args = defaultArgs.concat([
         'ssh', '--', 'ip', '-4', '-o', 'addr', 'show', 'dev', 'eth0'
-      ];
+      ]);
```

Comment:
Instead of the `docker-machine-driver-hyperkit ssh ...` command you could simply use `docker-machine-driver-hyperkit ip` to fetch the IP address out of the machine config file...

That IP address will be correct, otherwise `ssh` would fail, because that is the IP address it will ssh into 😄